packages/wdio-reporter/tests/reporter.test.ts

```diff
@@ -65,6 +65,13 @@ describe('WDIOReporter', () => {
         expect(options.writeStream.write).toBeCalledWith('foobar')
     })
 
+    it('should set contentPresent to true when content is passed to write()', () => {
+        const options = { stdout: true, writeStream: { write: jest.fn() } as unknown as WriteStream }
+        const reporter = new WDIOReporter(options)
+        reporter.write('foobar')
```

Comment:
Let's double check that the flag is false before
```suggestion
        expect(reporter.contentPresent).toBe(false)
        reporter.write('foobar')
```